src/App.js

```diff
@@ -779,17 +779,23 @@ export default class App extends Component {
     const {
       proposals: proposalsList,
       voteEndTime,
-      voteStartTime
+      voteStartTime,
+      trashAddress
     } = body.contents;
-    const proposals = proposalsList.filter(p => p.proposalId)
+
+    const proposals = proposalsList
+      .map((p,i)=>({...p, id: i }))
```

Comment:
So you're using position of proposal in original data as an id for vote and stuff (https://github.com/deora-earth/voting-frontend/pull/17/files#diff-ccfae9925ce47f02a6a2317f3cd933fbR98). What's wrong with `proposalId`? What will happen if position of proposal is changed?

Reply:
![proposal-id](https://user-images.githubusercontent.com/3136647/64315548-09b1ec00-cfbb-11e9-84b9-f17b290a759f.jpg)
Because of this 👆

Reply:
I see